pycytominer/cyto_utils/cells.py

```diff
@@ -397,6 +397,34 @@ def get_subsample(self, df=None, compartment="cells", rename_col=True):
 
         self.is_subset_computed = True
 
+    def is_feature_col(self, col):
```

Comment:
I made a comment about this function in the proposed test update.

I think that you can easily update this function to iterate through `self.compartments` rather than hardcoding canonical compartments here. (i.e. what about non-canonical compartments?)

Reply:
Some quick context here for @bunnech 

The so-called canonical -- or default -- compartments are `cells`, `cytoplasm`, and `nuclei` (and their corresponding column name prefixes are `Cells_`, `Cytoplasm_`, and `Nuclei_`, respectively).

But in general, it's possible to 
- have other ("non-canonical") compartments, and 
- (I think), not have some of these canonical compartments

Reply:
I compared with `self.compartments`. Thanks for the suggestion @gwaybio  and explanation @shntnu!